src/ape/api/accounts.py

```diff
@@ -401,15 +401,24 @@ class AccountContainerAPI(BaseInterfaceModel):
     instances.
     """
 
-    data_folder: Path
+    """
+    The name of the account container.
+    For example, the ``ape-ledger`` plugin
+    uses ``"ledger"`` as its name.
+    """
+    name: str
 
+    """
+    The type of account in this container.
```

Comment:
migrated some of the docs changes so we can get it in the next 0.7 release as well: https://github.com/ApeWorX/ape/pull/2026/files